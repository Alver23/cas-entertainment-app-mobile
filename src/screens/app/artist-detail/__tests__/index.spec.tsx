// Dependencies
import React from 'react';
import { shallow, ShallowRendererProps } from 'enzyme';
import * as ReactNavigationHooks from '@react-navigation/native';

// Under test file
import ArtistDetail from '../index';

// Mocks
jest.mock('@react-navigation/native');

describe('<ArtistDetail />', () => {
  let component: ShallowRendererProps;
  const navigation: any = {
    params: { id: 1 },
  };

  beforeEach(() => {
    jest.spyOn(ReactNavigationHooks, 'useRoute').mockReturnValue(navigation);
    component = shallow(<ArtistDetail />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
