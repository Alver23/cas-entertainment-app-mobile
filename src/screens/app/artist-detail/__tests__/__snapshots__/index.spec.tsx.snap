// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArtistDetail /> should save a snapshot of the component 1`] = `
<LayoutView
  style={
    Object {
      "flex": 1,
      "paddingHorizontal": 16,
    }
  }
>
  <ArtistDetail
    id={1}
  />
</LayoutView>
`;
